HexCtrl/src/Dialogs/CHexDlgDataInterpret.cpp

```diff
@@ -15,6 +15,16 @@
 using namespace HEXCTRL;
 using namespace HEXCTRL::INTERNAL;
 
+//Time calculation constants
+//
+#define FTTICKSPERMS				10000			//Number of 100ns intervals in a milli-second
+#define FTTICKSPERSECOND			10000000		//Number of 100ns intervals in a second
+#define HOURSPERDAY					24				//24 hours per day
+#define SECONDSPERHOUR				3600			//3600 seconds per hour
+#define FILETIME1582OFFSETDAYS		6653			//FILETIME is based upon 1 Jan 1601 whilst GUID time is from 15 Oct 1582. Add 6653 days to convert to GUID time
+#define FILETIME1970_LOW			0xd53e8000		//1st Jan 1970 as FILETIME
+#define FILETIME1970_HIGH			0x019db1de		//Used for Unix and Java times

```

Comment:
Please, avoid using macros for constants by any means.
In c++ >= C++11 we have `constexpr` for that.
Macros are only for conditional compiling.